### DISCUSSION TITLE
Buff T4 Raids

### DISCUSSION THREAD

**Discussion by shaszun** [chromiecraft/chromiecraft#5924] [Suggestions]
Hello,

My suggestion is to buff the just released Gruul's Lair and Magtheridon, more dmg and more hp.

Here are some suggestions.
Gruul's trash:
HP: 1M+
DMG: x1.5+

High King's Guards:
HP: 1M+
DMG: I didn't have the chance to observe all, but Kiggler looked the hardest hitter there. Needs boost if not all, most of them.

High King Maulgar:
HP: 3M+
DMG: Big revision is requred, we had whirlwind with all DPS staying there, they barely took 10k total overall between all of them. This should be a curtain dead if you are not a tank.

Gruul:
HP: 10M+  We killed it with 3 stacks without bloodlust(it was on CD from High King)
DMG: Explosion dmg looked fine. Everything else was a joke.

Magtheridon trash:
HP: x2-3
DMG: Looked good, maybe x1.5

Magtheridon Guards:
HP: 300k is too low, we killed them with 50 seconds to spare for the boss' banish to expire. Suggestion x4
DMG: Needs a lot of boosting, raid barely took any dmg.

Magtheridon: 
HP: x3
DMG: Hit & Cleave might need some boost. Everything else looked decent.
Cubes: I don't think they work properly and stop the blast wave (I might be wrong and someone fucked it up in the raid). If you have enough healers you more or less can skip the clicking and just outheal as much as you can and zurg the boss after. Also people said the debuff is like 30 seconds. This is way short, normally you need like 4 clickers for each cube to kill the boss with current gear.

**Comment by 55Honey** [comment]
Hello shaszun,
thanks for reaching out.
I take it you're not aware of our planned progression with multiple difficulties. Feel free to catch up with our [discord](https://discord.com/channels/788063059926712341/788389203955941408/1127720673679261716) or the same link on our [website](https://www.chromiecraft.com/en/mythic-dungeons-raids-release-schedule/).

As for bug reports, these require blizz-like sources and don't belong into discussions, but issues instead.

Regards

**Reply by shaszun** [reply]
I'm aware with the mythics. Half the guild complained that it's way too easy and are dissapointed, skipping more or less all tactics in the fights. Also a lot of my guildmates don't want custom stuff, they just want hard standard raids. Where they had nice memories from years ago, you know the nostalgia they expirienced when they raided if for first time.

**Reply by 55Honey** [reply]
It's obvious that we can not possibly please any and all players with an approach. We went the other route this time.

**Reply by lord-spirit** [reply]
Why not both?
1. Normal (as they are now for pugs and lazy people that just want loot)
2. Heroic (tuned like Vanilla raids were to be an actual challenge)
3. Mythic (Custom difficulty, custom mechanics, custom rewards)

**Reply by Peppersprayz** [reply]
I can understand easy version of the raids for casual/PUG.
I can understand mythic version of the raids with custom mechanics for tryharders.
I can understand a AQ40-like tuning of the raids for people that want challenging but "blizzlike" raids.
What i can't understand is why some people seems to think that you can't please everyone whereas it would actually be possible to just have those 3 raid versions instead of just the 2 firsts.
